Tests/test_PDB.py

```diff
@@ -1231,14 +1257,14 @@ def test_entitity_copy(self):
 
 
 def eprint(*args, **kwargs):
-    '''Helper function that prints to stderr.'''
+    """Helper function that prints to stderr."""
     print(*args, file=sys.stderr, **kwargs)
 
 
 def will_it_float(s):
-    '''
+    """
```

Comment:
Thanks for making this docstring (and the others) use the triple-double-quotes style, that's great. Could you also fix this entry to have a blank line after the one-line summary - see PEP257 https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings